Gemfile

```diff
@@ -88,6 +88,7 @@ end
 group :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
+  gem 'fakeredis', require: 'fakeredis/rspec'
```

Comment:
Another option is to actually use Redis in tests. I don't have an opinion about this... LGTM

Reply:
You should add the version `0.8.0`

Reply:
IMO for testing is ok not to use a real instance of redis

Reply:
Yeah, but one could argue, why not doing the same with Postgres?

Reply:
maybe because of the relationships that exist in the DB, while Redis is only a cache-like memory DB where there are no relationships